packages/evolution-frontend/src/components/inputs/__tests__/InputRadio.test.tsx

```diff
@@ -214,5 +214,51 @@ describe('Render InputRadio with minimum parameters', () => {
         );
         expect(wrapper).toMatchSnapshot();
     });
-    
+});
+
+describe('Render InputRadio with HTML label', () => {
+
+    const choices = [
+        {
+            value: 'first value',
+            label: {
+                fr: '<div>premiere valeur</div>',
+                en: '<div>first value</div>'
+            }
+        },
+        {
+            value: 'second value',
+            label: '<div>second value</div>'
```

Comment:
What if you inject javascript code in this label? What would it look like in the snapshot?

Reply:
So I tried on a test project, and it seems like a function is rendered as as normal string :
<img width="564" alt="image" src="https://github.com/chairemobilite/evolution/assets/80251305/7b256001-6990-4db8-b96e-506a36905369">
<img width="254" alt="image" src="https://github.com/chairemobilite/evolution/assets/80251305/90bbd0be-b4f0-4a4d-8280-e65643c3b5f1">

However, if we were to pass a value to the function (adding a () at the end of the function use) : 
<img width="580" alt="image" src="https://github.com/chairemobilite/evolution/assets/80251305/50993914-b45e-4437-8787-4f46af80facc">
we can see that the function is called : 
<img width="501" alt="image" src="https://github.com/chairemobilite/evolution/assets/80251305/f62fae54-57f5-438a-bcd5-bedfc7e4c97f">

My guess is that it works the same way as an `onClick` prop, meaning that if you pass a value instead of the function, it will be rendered on the first page load as described here : https://sentry.io/answers/why-can-t-the-react-js-onclick-event-pass-a-value-to-a-method/

Moreover, I am only using js on my test project so there is no type checking. But EVOLUTION is using ts, meaning that passing some js code would throw a type error : 
<img width="1205" alt="image" src="https://github.com/chairemobilite/evolution/assets/80251305/1ff94226-30cb-498c-9fb5-f3062bd6ee41">